packages/bap-analyze/bap-analyze.2.2.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+name: "bap-analyze"
+version: "2.2.0"
+maintainer: "Ivan Gotovchits <ivg@ieee.org>"
+authors: "BAP Team"
+homepage: "https://github.com/BinaryAnalysisPlatform/bap/"
+bug-reports: "https://github.com/BinaryAnalysisPlatform/bap/issues"
+dev-repo: "git://github.com/BinaryAnalysisPlatform/bap/"
+license: "MIT"
+build: [
+  ["./configure" "--prefix=%{prefix}%" "--enable-analyze"]
+  [make]
+]
+install: [[make "install"]]
+remove: [["ocamlfind" "remove" "bap-plugin-analyze"]
+         ["bapbundle" "remove" "analyze.plugin"]
+         ]
+depends: [
+  "ocaml" {>= "4.08.0" }
+  "core_kernel" {>= "v0.14" & < "v0.15"}
+  "ppx_bap" {>= "v0.14" & < "v0.15"}
+  "bap-std" {= "2.2.0"}
+  "monads"  {= "2.2.0"}
+  "bap-knowledge" {= "2.2.0"}
+  "bap-core-theory" {= "2.2.0"}
+  "bap-main" {= "2.2.0"}
+  "bitvec" {= "2.2.0"}
```

Comment:
I think I told you before, but just case that helps you, this kind of pattern can be replaced by `{= version}`

Reply:
Yep, it is just how our release tools are implemented. They take the latest tag and substitute `master` for the version, everywhere (using good old `sed`). But we will probably switch to `version` later.